robottelo/vm.py

```diff
@@ -265,21 +266,34 @@ def register_contenthost(self, org, activation_key=None, lce=None,
 
         :param activation_key: Activation key name to register content host
             with.
+        :param lce: lifecycle environment name to which register the content
+            host.
         :param org: Organization name to register content host for.
         :param force: Register the content host even if it's already registered
         :param releasever: Set a release version
+        :param user_name: a user name to register the content host with
+        :param user_password: the user password
         :return: SSHCommandResult instance filled with the result of the
             registration.
 
         """
+        if ((user_name is None and user_password is not None)
```

Comment:
that seems as redundant verification as it obvious fact that you need to provide both parameters. 'dump user' checks is not needed for our project